src/index.js

```diff
@@ -52,9 +51,10 @@ function Debug(options) {
   }
 
   var config = {
+    projectIdRequired: false,
     baseUrl: 'https://clouddebugger.googleapis.com/v2',
     scopes: [
-      // TODO: do we still need cloud-platform scope? 
+      // TODO: do we still need cloud-platform scope?
```

Comment:
Yep, that's why the TODO is there. I don't want to change this as part of this PR though.